packages/scripts/README.md

```diff
@@ -368,14 +376,14 @@ This is how you execute the script with presented setup:
 -   `npm start` - starts the build for development.
 -   `npm run start:hot` - starts the build for development with "Fast Refresh". The page will automatically reload if you make changes to the files.
 -   `npm run start:custom` - starts the build for development which contains two entry points and a custom output directory. Paths for custom entry points are relative to the project root.
--   `npm run start:copy-php` - starts the build for development and opts into copying PHP files from the `src` directory and its subfolders to the output directory.
+-   `npm run start:copy-php` - starts the build for development and opts into copying all PHP files from the `src` directory and its subfolders to the output directory. By default, only PHP files listed in the `render` field in the detected `block.json` files get copied.
```

Comment:
> By default, only PHP files listed in the `render` field in the detected `block.json` files get copied.

As I understand from https://github.com/WordPress/gutenberg/pull/42430 the new `render` field in `block.json` only accepts a path for just one file.

Shouldn't this say something like...
> By default, only the PHP file defined for the `render` field in the detected `block.json` get copied.

Reply:
Yes, I see what you mean. It's hard to explain as it's one file per `render,` but it might be in multiple `block.json` files processed.

Reply:
Same as https://github.com/WordPress/gutenberg/pull/43917#discussion_r966939998